server/src/test/java/org/elasticsearch/action/bulk/BulkRequestTests.java

```diff
@@ -405,7 +405,7 @@ public void testInvalidDynamicTemplates() {
             IllegalArgumentException.class,
             () -> new BulkRequest().add(updateWithDynamicTemplates, null, XContentType.JSON)
         );
-        assertThat(error.getMessage(), equalTo("Update request in line [2] does not accept dynamic_templates"));
+        assertThat(error.getMessage(), equalTo("Update request in line [1] does not accept dynamic_templates"));
```

Comment:
The way that this worked before, the parser would improperly identify the line that the error occurred on.